app/src/main/java/com/artemzin/qualitymatters/developer_settings/DevMetricsProxy.java

```diff
@@ -0,0 +1,8 @@
+package com.artemzin.qualitymatters.developer_settings;
+
+public interface DevMetricsProxy {
+  /**
+   * Applies performance metrics library
```

Comment:
http://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

Needs `.` in the end :)